bindings/matrix-sdk-crypto-ffi/src/lib.rs

```diff
@@ -925,6 +925,63 @@ fn vodozemac_version() -> String {
     vodozemac::VERSION.to_owned()
 }
 
+/// The encryption component of PkEncryption support.
+///
+/// This struct can be created using a [`Curve25519PublicKey`] corresponding to
+/// a `PkDecryption` object, allowing messages to be encrypted for the
+/// associated decryption object.
+#[derive(uniffi::Object)]
+pub struct PkEncryption {
+    inner: matrix_sdk_crypto::vodozemac::pk_encryption::PkEncryption,
+}
+
+#[uniffi::export]
+impl PkEncryption {
+    /// Create a new [`PkEncryption`] object from a `Curve25519PublicKey`
+    /// encoded as Base64.
+    ///
+    /// The public key should come from an existing `PkDecryption` object.
+    /// Returns a `DecodeError` if the Curve25519 key could not be decoded
+    /// correctly..
```

Comment:
nit: `..` at the end.

Reply:
Thanks, fixed in https://github.com/matrix-org/matrix-rust-sdk/pull/3971/commits/1fa5132f4b7902cec222abab0815a80787ef6763.